css/registrationForm.css

```diff
@@ -0,0 +1,23 @@
+form {
+ background-color: #f4f5f0;
+ width: 250px;
+ border: 1px solid;
+}
+h4 {
+    background-color:  #97c00e;
+    text-align: center;
+    color: white;
+    padding: 7px;
+    margin: -1px -1px 20px -1px;
+}
+.fieldsStyle{
```

Comment:
If you would like to improve further you may want to read on some conventions, e.g.: http://bdavidxyz.com/blog/how-to-name-css-classes/